sites/platform/src/create-apps/app-reference.md

```diff
@@ -273,16 +273,32 @@ For more information, see how to [manage resources](/manage-resources.md).
 
 ## Mounts
 
-Mounts define directories that are writable after the build is complete.
-They aren't available during the build.
+After your app is built, its file system is read-only.
+You need to use Git as the principal way to make changes to your app.
+However, you may not want to rebuild and redeploy your app every time you need to make a small change to it.
```

Comment:
It's not actually about making changes to the application. This must go through Git.
It's about writing files generated by the application (uploaded files, writing cache, writing logs, etc)